src/main/java/net/testudobank/MvcController.java

```diff
@@ -411,7 +416,7 @@ public String submitDispute(@ModelAttribute("user") User user) {
         // fetch updated overdraft balance with extra interest rate applied
         double updatedOverdraftBalanceInPennies = TestudoBankRepository.getCustomerOverdraftBalanceInPennies(jdbcTemplate, userID);
         // reverse extra application of interest rate since customer was already in overdraft
-        int newOverdraftBalanceInPennies = (int) (updatedOverdraftBalanceInPennies / 1.02);
+        int newOverdraftBalanceInPennies = applyInterestRateToPennyAmount((int)updatedOverdraftBalanceInPennies);
```

Comment:
This section is supposed to DE-apply the interest rate, so you shouldn't be using your new helper method here.